.github/workflows/dojot_acl.yml

```diff
@@ -0,0 +1,113 @@
+name: Dojot ACL (connector/dojot_acl)
+
+# Triggered after a branch is created, a tag or a branch is pushed or a PR with changes in
+# 'connector/dojot_acl/**' is submitted
+on:
+  create:
+  push:
+  pull_request:
+    paths:
+      - 'connector/dojot_acl/**'
+
+jobs:
+  unit-tests:
+    # Path to help identify each job in the status checks
+    name: (connector/dojot_acl) Unit Tests
+    # If the commit message contains 'ci skip', it is not necessary to run this job
+    if: contains(github.event.head_commit.message, 'ci skip') == false
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, build, lint and test
+      run: |
+        cd connector/dojot_acl
+        npm ci
+        npm run build --if-present
+        npm run lint
+        npm test
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        # Created when 'npm test' was executed
+        file: connector/dojot_acl/coverage/lcov.info
+        # Flags allow you to isolate coverage reports for different tests
+        flags: dojotACL
+        # Custom name for the upload
+        name: dojot-acl
+
+  docker-hub:
+    needs: unit-tests
+    # Path to help identify each job in the status checks
+    name: (connector/dojot_acl) Build and Publish image to DockerHUb
+    runs-on: ubuntu-latest
+    # If this is a PR, it won't run
+    if: contains(github.ref, 'refs/pull/') == false
+    steps:
+
+    - name: Owner Repository
+      id: get_owner
+      run: |
+        export OWNER=$(echo ${{ github.repository }} | cut -f1 -d"/")
+        echo Current Owner is $OWNER
+        echo ::set-output name=owner::$OWNER
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Build and Push
+      uses: elgohr/Publish-Docker-Github-Action@2.16
+      with:
+        name: ${{ steps.get_owner.outputs.owner }}/dojot-acl
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        workdir: connector/dojot_acl
+        # Label with the built image's commit
+        buildoptions: " --label 'commit_sha'='${{ github.sha }}' "
+        # Whether tag will be based in the branch name or in the tag name
+        tag_names: true
+
+  gh-pages:
```

Comment:
I think you can remove the gh-pages step, because there's no api documentation in this service right now.